recognition/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# syntax = docker/dockerfile:1.2
+
+FROM python:3.10
+
+WORKDIR /home/revue
+
+COPY .env .env
+COPY recognition recognition
+
+WORKDIR /home/revue/recognition
+
+# Install yolov3 neural network weights
+RUN wget -N -P app/resources/yolov3/ -c "https://pjreddie.com/media/files/yolov3.weights" --header "Referer: pjreddie.com"
```

Comment:
🥇 🥇

Reply:
This stuff takes 5 min to be downloaded ⚙️ ☠️